ws/websocket.go

```diff
@@ -282,6 +283,7 @@ func NewServer() *Server {
 // If no tlsConfig parameter is passed, the server will by default
 // not perform any client certificate verification.
 func NewTLSServer(certificatePath string, certificateKey string, tlsConfig *tls.Config) *Server {
+	router := mux.NewRouter()
```

Comment:
With the new logic the router needs to be created in normal mode (non-TLS) as well, otherwise starting the server will panic. There's a `NewServer()` function a few lines above which would basically need the same addition.

Reply:
Sorry, I added this fix here 🙇 
https://github.com/lorenzodonini/ocpp-go/pull/180/commits/85e04dd7eb82a86da294a73caf3e8e6c9a4d02cb